### ISSUE TITLE
mostEmotionValue error

### ISSUE THREAD

**Issue by chenfengfeng** [Microsoft/Cognitive-Face-iOS#18]
e.g:
```
// self.mostEmotionValue  = 0,self.happiness = 0.464, self.sadness = 0.014
        if (self.happiness > self.mostEmotionValue)
        {
            self.mostEmotion = @"happiness";
            self.mostEmotionValue = self.happiness;
        }
        if (self.sadness > self.mostEmotionValue)
        {
            self.mostEmotion = @"sadness";
            self.mostEmotionValue = self.sadness;
        }

self.mostEmotion = @"sadness";
self.mostEmotionValue = 0.014;
```

**Comment by huxuan** [comment]
Hi @chenfengfeng, Thanks for your report. Actually, I think it is already fixed by commit de109c36f5566881426744adcc8d35ec89b061c7 which is included in pull reqeust #17, but we still need some process for release. I will keep this issue open until the new version is released.

**Comment by chenfengfeng** [comment]
👌

**Comment by huxuan** [comment]
Fixed by #24 and should be available in version 1.2.6 (on cocoapods).
